doc/source/f2py/f2py.getting-started.rst

```diff
@@ -6,52 +6,55 @@ Wrapping Fortran or C functions to Python using F2PY consists of the
-  functions. In the case of Fortran routines, F2PY can create initial
+  wrappers to Fortran or C functions, also called the signatures of the
+  functions. For Fortran routines, F2PY can create an initial
   signature file by scanning Fortran source codes and
-  catching all relevant information needed to create wrapper
+  tracking all relevant information needed to create wrapper
   functions.
 
-* Optionally, F2PY created signature files can be edited to optimize
-  wrappers functions, make them "smarter" and more "Pythonic".
+  * Optionally, F2PY created signature files can be edited to optimize
+    wrapper functions, to make them "smarter" and more "Pythonic".
 
 * F2PY reads a signature file and writes a Python C/API module containing
   Fortran/C/Python bindings.
-
 * F2PY compiles all sources and builds an extension module containing
-  the wrappers. In building extension modules, F2PY uses
-  ``numpy_distutils`` that supports a number of Fortran 77/90/95
-  compilers, including Gnu, Intel,
-  Sun Fortre, SGI MIPSpro, Absoft, NAG, Compaq etc. compilers.
+  the wrappers.
+
+  * In building the extension modules, F2PY uses ``numpy_distutils`` which
+    supports a number of Fortran 77/90/95 compilers, including Gnu, Intel, Sun
+    Fortran, SGI MIPSpro, Absoft, NAG, Compaq etc.
+
+Depending on the situation, these steps can be carried out in a single composite
+command or step-by-step; in which case some steps can be omitted or combined
+with others.
 
-Depending on a particular situation, these steps can be carried out
-either by just in one command or step-by-step, some steps can be
-omitted or combined with others.
+Below we describe three typical approaches of using F2PY. These can be read in
+order of increasing effort, but also cater to different access levels depending
+on whether the Fortran code can be modified freely.
 
-Below I'll describe three typical approaches of using F2PY.
 The following example Fortran 77 code will be used for
-illustration, save it as fib1.f:
+illustration, save it as ``fib1.f``:
 
-.. include:: fib1.f
-   :literal:
+.. literalinclude:: ./code/fib1.f
+   :language: fortran
 
 
 The quick way
 ==============
 
-The quickest way to wrap the Fortran subroutine ``FIB`` to Python is
+The quickest way to expose the Fortran subroutine ``FIB`` to Python is
 to run
 
 ::
 
   python -m numpy.f2py -c fib1.f -m fib1
 
-This command builds (see ``-c`` flag, execute ``python -m numpy.f2py`` without
-arguments to see the explanation of command line options) an extension
-module ``fib1.so`` (see ``-m`` flag) to the current directory. Now, in
-Python the Fortran subroutine ``FIB`` is accessible via ``fib1.fib``::
+This command builds (see the ``-c`` flag, or execute ``python -m numpy.f2py``
+without arguments to see the explanation of command line options) an extension
+module ``fib1.so`` (see the ``-m`` flag) to the current directory. Now, in Python
+the Fortran subroutine ``FIB`` is accessible via ``fib1.fib``::
```

Comment:
Maybe 

"This command compiles and wraps ``fib1.f`` (``-c``)  to create the extension module ``fib1.so`` (``-m``)  in the current directory.  A list of command line options can be seen by executing ``python -m numpy.f2py``."